meta-mbl-distro/scripts/create-update-payload

```diff
@@ -64,6 +66,38 @@ sys.path.insert(0, str(mbl_lib_path))
 
 import mbl.update.payload as upayload  # noqa: E402
 
+# list of mbl-os versions in which the payload format changed, along with the
+# format version used
+FORMAT_CHANGES = (("0.9.0", 1), ("0.10.0", 3))
+
+
+def _str_remove_prefix(orig_str, prefix):
+    if orig_str.startswith(prefix):
+        return orig_str[len(prefix) :]
+    return orig_str
+
+
+def _payload_format_version_for_os_version(os_version_str):
+    parsed_format_changes = sorted(
+        (pkg_resources.parse_version(x[0]), x[1]) for x in FORMAT_CHANGES
```

Comment:
I think we should use tuple unpacking here for readability. You should be able to do something along the lines of
```
for elem_a, elem_b  in FORMAT_CHANGES
```

Reply:
Okay: https://github.com/ARMmbed/meta-mbl/pull/792/commits/e617144decfebe45c9ad87bfd022634043a21646